src/server/api/schema.js

```diff
@@ -668,7 +668,7 @@ const rootMutations = {
       campaigns.forEach(campaign => { campaign.is_archived = true })
       await Promise.all(campaigns.map(campaign => {
         campaign.save()
-        cacheableData.campaign.clear(id)
+        cacheableData.campaign.clear(ids)
```

Comment:
I think if ids is an array, we need to do a `for (let i=0,l=ids.length; i<l; i++) { cacheableData.campaign.clear(id) }`

Reply:
https://github.com/MoveOnOrg/Spoke/blob/6f82c3f94f1b654653d9f7093bc69a7712e74f66/src/server/models/cacheable_queries/campaign.js#L50